src/utils.ts

```diff
@@ -0,0 +1,19 @@
+import * as logger from './logger';
+import * as core from '@actions/core'
+
+export const WORKFLOW_TELEMETRY_SERVER_PORT = "WORKFLOW_TELEMETRY_SERVER_PORT";
+
+export const WORKFLOW_TELEMETRY_VERSIONS = {
+    METRIC: "v1",
+    PROCESS: "v1"
+};
+
+export async function setServerPort() {
+    var portfinder = require('portfinder');
```

Comment:
It is better to specify base port. By default it starts from `8000` and we don't want to use a common port here. So lets set `basePort` (https://www.npmjs.com/package/portfinder#ports-search-scope) to a number something like `10000`